opennms-doc/guide-development/src/asciidoc/text/rest/remotepoller.adoc

```diff
@@ -0,0 +1,59 @@
+
+==== Remote Roller
+
+===== GETs (Reading Data)
+
+Allows to query the aggregated percentage availability for a given specific application and its services.
+
+NOTE: This API is v2-only
+
+[options="header", cols="5,10"]
+|===
+| Resource                                           | Description
+| `remotepoller/{applicationId}`                     | Get the list of aggregated percentage availability per location for the application specified by _applicationId_.
```

Comment:
Would it make sense to call it `/remotepoller/availability/{applicationId}` and `/remotepoller/availability/{applicationId}`? That way don't have to break the API later when we want to publish also other information from a remote poller application, like status or whatever we need to publish?

Reply:
I have currently implemented the API as defined in Confluence. I'm sure this will change over time, but at the moment the UI work is based on exactly this API.

Reply:
ok got it :)